proto/binlog/pump.proto

```diff
@@ -0,0 +1,31 @@
+syntax = "proto3";
+
+package binlog;
+
+import "gogoproto/gogo.proto";
+
+option (gogoproto.marshaler_all) = true;
+option (gogoproto.unmarshaler_all) = true;
+option (gogoproto.sizer_all) = true;
+
+// Interfaces exported by Pump.
+service Pump {
+    // Writes a binlog to the local file on the pump machine.
+    // A response with an empty errmsg is returned if the binlog is written successfully.
+    rpc WriteBinlog(WriteBinlogReq) returns (WriteBinlogResp) {}
+}
+
+message WriteBinlogReq {
+    // The identifier of tidb-cluster, which is given at tidb startup.
+    // Must specify the clusterID for each binlog to write.
+    uint64 clusterID = 1;
+
+    // Payload bytes can be decoded back to binlog struct by the protobuf.
+    bytes payload = 2;
+}
+
+message WriteBinlogResp {
+    // An empty errmsg returned means a successful write.
+    // Otherwise return the error description.
+    string errmsg = 1;
```

Comment:
It's a very simple RPC method, return a nil for success just like func in golang.
I think it's clear and simple enough in the early stage. 
However, when the caller needs to be treated according to the return value, an error code is necessary.